.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
It's a good idea to add `.vscode/` to your `gitignore` file, so you don't accidentally check in your own editor metadata.